projects/ngx-meta/src/core/src/utils/make-injection-token.spec.ts

```diff
@@ -8,12 +8,14 @@ import { InjectionToken } from '@angular/core'
 
 describe('make injection token', () => {
   const sut = _makeInjectionToken
-  const description = 'dummy'
+  const description = 'dummy-injection-token-for-unit-tests'
   const factory = () => description
+  // 👇 See test below about providing same description and factory
+  const sameFactoryDifferentIdentity = () => description
 
   afterEach(() => {
-    INJECTION_TOKENS.clear()
-    INJECTION_TOKEN_FACTORIES.clear()
+    INJECTION_TOKENS.delete(description)
+    INJECTION_TOKEN_FACTORIES.delete(description)
```

Comment:
Not actually needed. However nice to only mess one dummy injection token just in case.